index.html

```diff
@@ -18,26 +18,26 @@ <h1 class="header-title">Check <span class="yo">Yo'</span> Self</h1>
     </header>
     <section class="sidebar">
       <section class="sidebar-toDoCreator-inputsAndList">
-        <section class="sidebar-toDoCreator-taskInput">
-          <label for="task-title">Task Title</label>
+        <!-- <section class="sidebar-toDoCreator-taskInput"> -->
+          <label for="task-title" class="todoCreator-taskInput-textInputLabel">Task Title</label>
           <input type="text" name="task-title" class="todoCreator-taskInput-textInput"></input>
-        </section>
+        <!-- </section> -->
         <section class="sidebar-toDoCreator-taskList">
           <ul class="toDoCreator-taskList-fullList">
             <li class="toDoCreator-taskList-listItem">
-              <img src="./images/checkbox.svg"></img>
+              <img src="./images/checkbox.svg" class="toDoCreator-taskList-checkbox"></img>
               <p>Placeholder text</p>
             </li>
             <li class="toDoCreator-taskList-listItem">
-              <img src="./images/checkbox.svg"></img>
+              <img src="./images/checkbox.svg" class="toDoCreator-taskList-checkbox"></img>
               <p>Placeholder text</p></li>
             <li class="toDoCreator-taskList-listItem">
-              <img src="./images/checkbox.svg"></img>
+              <img src="./images/checkbox.svg" class="toDoCreator-taskList-checkbox"></img>
```

Comment:
Thanks for adding these classes, I was thinking of bringing that up tomorrow but glad you took care of it.